draft-ietf-dnsop-dns-capture-format-01.md

```diff
@@ -719,14 +723,12 @@ of the original packet stream cannot be re-constructed from the C-DNS format:
      * A DNS payload may have be split across multiple TCP segments
      * Multiple DNS messages may have be sent on a single TCP session
 * Malformed DNS messages if they are not recorded
-* Non-DNS packets
```

Comment:
But the above text is specifically describing trying to reconstruct the original PCAP from the C-DNS, so we really should mention everything that doesn't go into the C-DNS. If we need to clarify what we mean by Non-DNS, fine, but we can't just not mention it.....